website/contributing/how-to-build-from-source.md

```diff
@@ -45,13 +45,7 @@ ndk.dir=/Users/your_unix_name/android-ndk/android-ndk-r20b
 
 #### Download links for Android NDK
 
-1. [macOS (64-bit)](https://dl.google.com/android/repository/android-ndk-r20b-darwin-x86_64.zip)
-2. [Linux (64-bit)](https://dl.google.com/android/repository/android-ndk-r20b-linux-x86_64.zip)
-3. [Windows (64-bit)](https://dl.google.com/android/repository/android-ndk-r20b-windows-x86_64.zip)
-4. [Windows (32-bit)](https://dl.google.com/android/repository/android-ndk-r20b-windows-x86.zip)
-
-You can find further instructions on the [official page](https://developer.android.com/ndk/index.html).
-Or follow instructions to [download appropriate NDK from SDK manager](https://developer.android.com/studio/projects/install-ndk#specific-version).
+You can find the instructions to install the NDK on the [official Android NDK website](https://developer.android.com/studio/projects/install-ndk). In case you need a specific version (ex. you are working on an older version of React Native) you can refer to these instructions to install [download appropriate NDK from SDK manager](https://developer.android.com/studio/projects/install-ndk#specific-version).
```

Comment:
```suggestion
You can find the instructions to install the NDK on the [official Android NDK website](https://developer.android.com/studio/projects/install-ndk).
```

This should not be needed at all at this point.

NDK gets installed on demand on the first build as you specify a version. I'm not sure people should go and download specific versions

Reply:
yeah, I guess that the specific set of people that will need an older NDK will know where to look for it